ui/css/treeherder.css

```diff
@@ -259,12 +82,9 @@ th-watched-repo {
     overflow-y: auto;
 }
 
-.nav-panel-help-text {
-    color: #2b393f;
-    margin-bottom: 10px;
-}
-
-/* Options drop-downs*/
+/*
+ * General panels
+ */
```

Comment:
Replacement content marker for treeherder.css during the splitting process. Eventually the stuff below it will likely go into `treeherder-panels.css`.